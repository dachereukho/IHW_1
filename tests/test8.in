6
20
-8
1
2
2
-7