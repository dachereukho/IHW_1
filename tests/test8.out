-8 2 -7 20 1 2